### ISSUE TITLE
Win 10 22H2 Fails

### ISSUE THREAD

**Issue by Toadally-Tech** [AveYo/MediaCreationTool.bat#380]
Win 10 22H2 terminates unexpectedly using auto ISO.

**Comment by AndyChips** [comment]
Yup - me too. No explanation for the failure.

![image](https://github.com/user-attachments/assets/215078fc-fe67-4ec7-b9e4-51a7739cc763)

**Comment by RollupTito** [comment]
Hello ! Same problem for us too. Links seem allright in the .bat file, and other versions are working well. 
No idea where the error come from.

**Comment by AndyChips** [comment]
Yeah, and it looks like the project author has finished with it and disappeared. 
Such a shame.

**Comment by togwar** [comment]
**PROBLEM:**
Downloaded file located in C:\ESD\MCT\MediaCreationTool22H2.exe is 1KB....
Link in Line 166 is broken...

**SOLUTION**
Replace line 166 to this URL: https://go.microsoft.com/fwlink/?LinkId=2265055

`set "EXE=https://go.microsoft.com/fwlink/?LinkId=2265055"`

**SUGESTION**
Use these **linkid** to avoid future problems in the script, get them directly from the solution download page...

**Comment by AndyChips** [comment]
Thanks, but still getting the same problem:

![Image](https://github.com/user-attachments/assets/a66246f2-87d0-4ede-bd79-bc26034341de)

**Comment by togwar** [comment]
I do, and correct for me... but i make other changes.... 
My suggestion.... follow the fork "lzw29107"

This project, still sleeping for long time....

**Comment by molitar** [comment]
> I do, and correct for me... but i make other changes.... My suggestion.... follow the fork "lzw29107"
> 
> This project, still sleeping for long time....

How do you find that fork to follow?

**Comment by Nevidebla** [comment]
> I do, and correct for me... but i make other changes.... My suggestion.... follow the fork "lzw29107"
> 
> This project, still sleeping for long time....

Still error
